fe2-android/app/src/main/res/layout/digital_cash_send_fragment.xml

```diff
@@ -55,6 +55,7 @@
       android:layout_height="wrap_content"
       android:ems="10"
       android:hint="@string/digital_cash_amount"
+      android:text="@string/digital_cash_send_initial_amount"
```

Comment:
Should it be a hint ? I'm really not sure

Reply:
The hint is the grey placeholder behind the field which appears when it is empty. Doing as text and not as a hint allows us to pre-fill with a customs value, which is what we wanted to do in order to make testing easier.